src/uikit.cs

```diff
@@ -16257,6 +16358,12 @@ interface UIFocusEnvironment {
 		[iOS (10, 0)]
 		[Export ("preferredFocusEnvironments", ArgumentSemantic.Copy)]
 		IUIFocusEnvironment[] PreferredFocusEnvironments { get; }
+
+		[NoiOS]
+		[TV (11,0)]
+		[Export ("soundIdentifierForFocusUpdateInContext:")]
+		[return: NullAllowed]
+		string GetSoundIdentifier (UIFocusUpdateContext context);
```

Comment:
Better use `NSString`. In part because the smart enum will expose them as such, with `GetConstant`.
Also because we can't be 100% the return `string` can be used back in the API (in case Apple does a pointer check).